src/nbl/video/COpenGLESFunctionTable.h

```diff
@@ -121,6 +125,18 @@ class COpenGLESFunctionTable final : public COpenGL_FunctionTableBase
 		glTexture.pglActiveTexture(activeTex);
 	}
 
+	void extGlTextureView(GLuint texture, GLenum target, GLuint origtexture, GLenum internalformat, GLuint minlevel, GLuint numlevels, GLuint minlayer, GLuint numlayers) override
+	{
+		if (glesTexture.pglTextureViewOES)
+			glesTexture.pglTextureViewOES(texture, target, origtexture, internalformat, minlevel, numlevels, minlayer, numlayers);
+		else if (glesTexture.pglTextureViewEXT)
+			glesTexture.pglTextureViewEXT(texture, target, origtexture, internalformat, minlevel, numlevels, minlayer, numlayers);
+		else
+		{
+			os::Printer::log("None of texture view extensions for GLES are supported, cannot create texture view!\n", ELL_ERROR);
+		}
```

Comment:
if you require one of the extensions dont even bother logging when pointer not present (or at least wrap it in a fi DEBUG macro)